### PR TITLE
EventType is not yet part of the v1 eventing API

### PR DESCRIPTION
**Pull request by lenalebt**
## Proposed Changes 

I tried adding an EventType as proposed in the example - but `EventType` still is in `v1beta1`, although the example states it would be in `v1` already.

### CODE REVIEW

**Comment by lenalebt** [comment]
@googlebot I signed it!

**Comment by omerbensaadon** [comment]
Great catch, looks like it's showing that way in the spec.

File: docs/eventing/event-registry.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ kubectl get eventtype dev.knative.source.github.push-34cnb -o yaml
 Omitting irrelevant fields:
 
 ```yaml
-apiVersion: eventing.knative.dev/v1
+apiVersion: eventing.knative.dev/v1beta1
```
Comment (csantanapr) [inline docs/eventing/event-registry.md:63]: Thank you @lenalebt for the update, this is correct

**Pull request merged**